### PR TITLE
Add Makefile build to Getting Started

### PR DESCRIPTION
**Pull request by marcrittinghaus**
The current Getting Started page only describes how to setup and build Unikraft with `kraft`. However, sometimes it makes sense to work with a `Makefile` directly. This commit adds a description on how to do this.

### CODE REVIEW

File: content/en/docs/getting-started/_index.md [hunk 1]
```diff
@@ -13,6 +13,8 @@ The easiest way to get started with Unikraft is to use the command-line utility
 running Unikraft applications.  With `kraft` you can seamlessly create a build
 environment for your unikernel and painlessly manage dependencies for its build.
 
+As an alternative to `kraft` it is also possible to build Unikraft using a regular [`Makefile`](#building-unikraft-via-makefile).
```
Comment (StefanJum) [inline content/en/docs/getting-started/_index.md:16]:
```suggestion
As an alternative to `kraft` it is also possible to build Unikraft using a regular [`Makefile`](docs/getting-started/#building-unikraft-via-makefile).
```

File: content/en/docs/getting-started/_index.md [hunk 2]
```diff
@@ -103,3 +105,95 @@ For more information about all the steps behind `kraft up`, check [Creating an A
...
+Afterwards, we save and exit the configuration menu.
+
+You can then build the unikernel with:
+```bash
+$ make
+```
+Consider adding `-jX` with `X` being the number of CPUs in your machine to speed up the build process.
+
+When the build completes you will have a Unikraft-based kernel image (`build/app-helloworld_kvm-x86_64`) that you can start with QEMU/KVM using the `-kernel` option.
+See [here](../operations/plats/kvm.md) for further information on running the unikernel with KVM.
```
Comment (StefanJum) [inline content/en/docs/getting-started/_index.md:164]:
```suggestion
See [here](docs/operations/plats/kvm) for further information on running the unikernel with KVM.
```

**Review by StefanJum — CHANGES_REQUESTED** [review]
This is a great addition.
Except for the few broken links, all looks good.

**Comment by marcrittinghaus** [comment]
> This is a great addition. Except for the few broken links, all looks good.

Thanks for catching these errors @StefanJum! I have fixed them as you suggested.

**Comment by StefanJum** [comment]
> > This is a great addition. Except for the few broken links, all looks good.
> 
> Thanks for catching these errors @StefanJum! I have fixed them as you suggested.

Hi @marcrittinghaus, I think you forgot to add the file to the commit, the links did not change :)

**Comment by marcrittinghaus** [comment]
> Hi @marcrittinghaus, I think you forgot to add the file to the commit, the links did not change :)

Sorry! :neutral_face: This is the git version of forgetting the attachment in an email 🤦🏻‍♂️

**Review by StefanJum — APPROVED** [review]
All good now.

Reviewed-by: Stefan Jumarea <stefanjumarea02@gmail.com>

**Pull request merged**